src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -35,7 +35,7 @@ class PexCli(TemplatedExternalTool):
     name = "pex"
     help = "The PEX (Python EXecutable) tool (https://github.com/pantsbuild/pex)."
 
-    default_version = "v2.1.159"
+    default_version = "v2.1.161"
     default_url_template = "https://github.com/pantsbuild/pex/releases/download/{version}/pex"
     version_constraints = ">=2.1.135,<3.0"
```

Comment:
Although Pex doesn't break you, it does add new features. If / when Pants starts using those new features, you need to bump this lower bound to match the features Pants requires of Pex.

Reply:
Acknowledged.  Will bump the minimum to match whatever new feature I'm able to get to first.